### DISCUSSION TITLE
Quickest way to add audit timestamps to tables

### DISCUSSION THREAD

**Discussion by liaujianjie** [supabase/supabase#1844] [Questions] [answered]
What's the easiest way to add audit timestamps (e.g. `created_at`, `updated_at`) to our tables? Would I also need to hand-write the application logic or postgres triggers for such a behaviour?

I figured since such timestamps are a pretty common requirement in most apps, there would probably be a better (or recommended) way to implement such a feature—but I couldn't find anything on this.

**Comment by soedirgo** [comment, marked as answer]
Hey @liaujianjie, you can try using the "Automatically update timestamps" script in SQL > Getting started > Welcome for your `updated_at` column. For a `created_at` column, a timestamptz column with a default value of `NOW()` should be enough (you can do this from the table editor).

**Reply by liaujianjie** [reply]
Thank you, this worked perfect! The "Automatically update timestamps" script was exactly what I was looking for.
